infrastructure/src/main/java/org/corfudb/infrastructure/AbstractServer.java

```diff
@@ -34,26 +51,51 @@ public void sealServerWithEpoch(long epoch) {
         // Overridden in log unit to flush operations stamped with an old epoch
     }
 
+    @Deprecated
     public abstract boolean isServerReadyToHandleMsg(CorfuMsg msg);
 
+    /**
+     * Determine if the server is ready to handle a request.
+     * @param request The incoming request message.
+     * @return True if the server is ready to handle this request, and false otherwise.
+     */
+    public boolean isServerReadyToHandleReq(RequestMsg request) {
```

Comment:
I think its cleaner to just overload this and not introduce a new "Req" suffix naming scheme to differentiated the two paths. I would make this abstract and oberload it with `public abstract boolean isServerReadyToHandleMsg(RequestMsg request);`

Reply:
Done.